packages/client/src/types.ts

```diff
@@ -14,6 +14,9 @@ export type GetWorkflowExecutionHistoryRequest =
 export type DescribeWorkflowExecutionResponse =
   proto.temporal.api.workflowservice.v1.IDescribeWorkflowExecutionResponse;
 export type RawWorkflowExecutionInfo = proto.temporal.api.workflow.v1.IWorkflowExecutionInfo;
+export type RawCountWorkflowExecutions = proto.temporal.api.workflowservice.v1.ICountWorkflowExecutionsResponse;
```

Comment:
We have no need for a reexport of `RawCountWorkflowExecutions` and `RawCountWorkflowExecutionsAggregationGroup`

Reply:
Done.